src/test/java/com/unosquare/admin_core/back_end/helper/UnitTestHelper.java

```diff
@@ -0,0 +1,141 @@
+package com.unosquare.admin_core.back_end.helper;
+
+import com.unosquare.admin_core.back_end.dto.EmployeeDTO;
+import com.unosquare.admin_core.back_end.dto.EventDTO;
+import com.unosquare.admin_core.back_end.dto.EventMessageDTO;
+import com.unosquare.admin_core.back_end.entity.*;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+public class UnitTestHelper {
+
+    private static final LocalDate date = LocalDate.now();
+    private static final LocalDateTime dateTime = LocalDateTime.now();
+
+    public Event createEvent() {
```

Comment:
Might be best leaving this until the work on TS-103.

Also for that, you'd be better having separate files for setting up each different type of testing object